projects/angular/src/layout/nav/_header.clarity.scss

```
@@ -531,54 +307,34 @@
 
       // Last Dropdown Menu adjustments so that the menu has some space between itself and the right end of the browser window
       :last-child.dropdown-menu {
-        margin-right: variables.$clr_baselineRem_8px;
+        margin-right: tokens.$cds-global-space-5;
       }
     }
   }
 }
 
 @mixin header-search() {
-  .search-box, /* TODO: deprecated. Remove when the clarity css naming convention is implemented.*/
+  .search-box,
   .search {
     display: flex;
     align-items: center;
     flex: 0 0 auto;
-    max-width: variables.$clr_baselineRem_12;
+    max-width: mixins.baselinePx(12 * 24);
     padding: 0;
-    @include mixins.css-var(
-      height,
-      clr-header-height,
-      header-variables.$clr-header-height,
-      variables.$clr-use-custom-properties
-    );
+    height: header-variables.$clr-header-height;
     @include header-nav-appearance();
 
     & > .nav-icon {
-      margin: 0 variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_0_125 variables.$clr_baselineRem_1;
+      margin: 0 tokens.$cds-global-space-4 mixins.baselinePx(3) tokens.$cds-global-space-9;
     }
 
     label {
       display: inline-block;
-      @include mixins.css-var(
-        height,
-        clr-header-height,
-        header-variables.$clr-header-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-header-height,
-        header-variables.$clr-header-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        padding-left,
-        clr-header-search-padding-left,
-        variables.$clr_baselineRem_1,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(margin-left, clr-header-search-margin-left, 0, variables.$clr-use-custom-properties);
-      @include mixins.css-var(margin-right, clr-header-search-margin-right, 0, variables.$clr-use-custom-properties);
+      height: header-variables.$clr-header-height;
+      line-height: header-variables.$clr-header-height;
+      padding-left: header-variables.$clr-header-search-gap;
+      margin-left: header-variables.$clr-header-search-margin-left;
+      margin-right: header-variables.$clr-header-search-margin-right;
```

Does it make sense to keep `margin` properties?
Does it make sense to use `$clr-header-search-gap` as padding or to have it at all?

```suggestion
      padding-left: tokens.$cds-global-space-5;
      margin-left: tokens.$cds-global-space-5;
      margin-right: tokens.$cds-global-space-7;
```

---

Yes. They correspond to spec definitions and will be directly used as gap and margins after refactor.